ios/PurchasesHybridCommon/PurchasesHybridCommonIntegrationTests/StoreKitIntegrationTests.swift

```diff
@@ -49,10 +54,51 @@ class StoreKit1IntegrationTests: BaseIntegrationTests {
         return .disabled
     }
 
+    func testCanGetOfferings() async throws {
+        let offerings = try await CommonFunctionality.offerings()
+
+        await self.assertSnapshot(offerings)
+    }
+
+    func testCanMakePurchase() async throws {
+        var data = try await self.purchaseMonthlyOffering()
+        removeDates(&data)
+
+        await self.assertSnapshot(data)
+    }
+
+    func testPurchaseFailuresAreReportedCorrectly() async throws {
+        self.testSession.failTransactionsEnabled = true
+        self.testSession.failureError = .invalidSignature
+
+        do {
+            try await self.purchaseMonthlyOffering()
+            fail("Expected error")
+        } catch {
+            expect(error).to(matchError(ErrorCode.invalidPromotionalOfferError))
+        }
+    }
+
+    func testLogInAndLogOut() async throws {
+        // Fetch receipt to make sure there aren't race conditions
```

Comment:
let's add another line here like
```swift
// since the receipt might not be available on device on the first run, 
// Fetch receipt to make sure there aren't race conditions dependent on the order of test runs
```

Reply:
Thanks! That's much clearer